Algorithm.CSharp/IndustryStandardSecurityIdentifiersRegressionAlgorithm.cs

```diff
@@ -0,0 +1,108 @@
+/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System.Collections.Generic;
+
+using QuantConnect.Interfaces;
+
+namespace QuantConnect.Algorithm.CSharp
+{
+    /// <summary>
+    /// Algorithm illustrating how to get a security's industry-standard identifier from its <see cref="Symbol"/>
+    /// </summary>
+    public class IndustryStandardSecurityIdentifiersRegressionAlgorithm : QCAlgorithm, IRegressionAlgorithmDefinition
+    {
+        public override void Initialize()
+        {
+            SetStartDate(2013, 10, 07);
+            SetEndDate(2013, 10, 07);
+
+            var spy = AddEquity("SPY").Symbol;
+            Log($"\nSPY CUSIP: {spy.CUSIP}" +
+                $"\nSPY Composite FIGI: {spy.CompositeFIGI}" +
+                $"\nSPY SEDOL: {spy.SEDOL}" +
+                $"\nSPY ISIN: {spy.ISIN}");
```

Comment:
We should probably assert these aren't empty/null? Think we can also add the `QCAlgorithm.ISIN(symbol)/etc` APIs here and assert their response are the same too 👌  same for the py algo